pkgs/top-level/all-packages.nix

```
@@ -17667,6 +17667,10 @@ with pkgs;
 
   rolespec = callPackage ../development/tools/misc/rolespec { };
 
+  rome = callPackage ../development/tools/rome {
+    inherit (darwin.apple_sdk.frameworks) Security;
+  };
```

```suggestion
  rome = callPackage ../development/tools/rome { };
```

Build currently fails since `Security` is an unexpected argument after your recent changes:

```console
error: anonymous function at /home/felschr/.cache/nixpkgs-review/pr-205414-1/nixpkgs/pkgs/development/tools/rome/default.nix:1:1 called with unexpected argument 'Security'
```

---

:sweat_smile: Ughh, missed it. Fixed now.